### PR TITLE
Yener spawn objects

### PR DESCRIPTION
**Pull request by yettyo**

### CODE REVIEW

File: Assets/Scripts/Spawner.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using UniRx;
+using System;
+using Lean.Pool;
+
+public class Spawner : MonoBehaviour
+{
+    private Collider SpawnArea;
+    private IDisposable SpawnDisposable;
+
+    public GameObject[] RingPrefabs;
```
Comment (yigitkalan) [inline Assets/Scripts/Spawner.cs]: in here if it doesn't have to be accessible from other classes variables should be private with [SerializeField] before them so we can edit the value in inspecter while keeping the data safe and encapsulated

**Review by yigitkalan — COMMENTED** [review]
in here if it doesn't have to be accessible from other classes variables should be private with [SerializeField] before them so we can edit the value in inspecter while keeping the data safe and encapsulated in spawner.cs file

File: Assets/Scripts/Spawner.cs (hunk 1 above)
Comment (yettyo) [reply Assets/Scripts/Spawner.cs]: Changed RingPrefab to private.

**Review by yigitkalan — DISMISSED** [review]
Okkayyy

**Review by yigitkalan — APPROVED** [review]
okay to merge

**Pull request merged**